spec/services/coronavirus/pages/sub_section_processor_spec.rb

```diff
@@ -1,73 +1,143 @@
 require "rails_helper"
 
 RSpec.describe Coronavirus::Pages::SubSectionProcessor do
-  let(:title) { Faker::Lorem.sentence }
-  let(:label) { Faker::Lorem.sentence }
-  let(:description) { Faker::Lorem.sentence }
-  let(:url) { "/#{File.join(Faker::Lorem.words)}" }
-  let(:label_1) { Faker::Lorem.sentence }
-  let(:url_1) { "/#{File.join(Faker::Lorem.words)}?priority-taxon=774cee22-d896-44c1-a611-e3109cce8eae" }
-  let(:data) do
-    [
-      {
-        "title" => title,
-        "list" => [
-          {
-            "label" => label,
-            "url" => url,
-            "featured_link" => true,
-            "description" => description,
-          },
-          {
-            "label" => label_1,
-            "url" => url_1,
-          },
-        ],
-      },
-    ]
-  end
-
   describe ".call" do
-    subject { described_class.call(data) }
-    let(:lines) { subject[:content].split("\n") }
+    it "doesn't add featured links to the lines of content" do
+      sub_section_payload_data = [
+        {
+          "title" => "Title",
+          "list" => [
+            {
+              "label" => "Label",
+              "url" => "/path",
+              "featured_link" => true,
+              "description" => "description",
+            },
+            {
+              "label" => "Another label",
+              "url" => "/another-path",
+            },
+          ],
+        },
+      ]
+
+      processor = described_class.call(sub_section_payload_data)
+      lines = processor[:content].split("\n")
 
-    it "creates the correct number of lines" do
-      # 2 = 1 title plus 1 link - feature_link's aren't created
       expect(lines.count).to eq(2)
```

Comment:
It might be more explicit to assert on the actual markdown here, I'm not convinced knowing that lines.count equals 2 is enough to know to be sure the featured link isn't in those lines? bit nit-picky I know.